src/SASViyaApiClient.ts

```diff
@@ -51,6 +51,16 @@ export class SASViyaApiClient {
   )
   private folderMap = new Map<string, Job[]>()
 
+  /**
+   * A helper method used to call appendRequest method of RequestClient
+   * @param response - response from sasjs request
+   * @param program - name of program
+   * @param debug - a boolean that indicates whether debug was enabled or not
+   */
+  public appendRequest(response: any, program: string, debug: boolean) {
```

Comment:
@YuryShkoda , I think name of this helper method should be changed as `RequestClient` class has a method with same name too. Also this method if the helper of that method. At some places we are appending `SASjsRequest` array but don't have access to `RequestClient` instance. Instead we have access to instance of `SASViyaApiClient` and this `SASViyaApiClient` instance have access to `RequestClient`. Therefore, I introduced this helper method.

Reply:
What should I name this method?